### ISSUE TITLE
Is there any methods being used to stop correlation between completed payments sent and browsing history?

### ISSUE THREAD

**Issue by kdenhartog** [WICG/webmonetization#443]
Currently it looks like the model of privacy has been focused on pre-payment flows and have done a good job of avoiding the Web monetization API from leaking browsing history. However, in the event that a payment is made via a more public system (thinking a publicly readable blockchain - although this may be out of scope) there's an opportunity for browsing history to be correlated to completed payments. Is this something that is considered within the scope of the threat model here or is there a way that it's being avoided currently?

**Comment by MichaelKohler** [comment]
> (thinking a publicly readable blockchain - although this may be out of scope)

Yes, that is indeed out of scope. However, you are bringing up a good point, we do need to keep in mind to update (and discuss) the threat model when we introduce changes.
